src/include/storage/write_ahead_log/log_manager.h

```diff
@@ -191,15 +191,15 @@ class LogManager {
   /**
    * Mark the current buffer that the serializer thread is writing to as filled
    */
-  void MarkBufferFull() {
-    filled_buffer_queue_.Enqueue(buffer_to_write_);
-    // Signal writer thread that a buffer is ready to be flushed to the disk
+  void HandFilledBufferToWriter() {
+    filled_buffer_queue_.Enqueue(hand_filled_buffer_);
+    // Signal consumer thread that a buffer is ready to be flushed to the disk
     {
       std::unique_lock<std::mutex> lock(persist_lock_);
-      wake_writer_thread_cv_.notify_one();
+      wake_consumer_thread_cv_.notify_one();
     }
     // Mark that serializer thread doesn't have a buffer in its possession to which it can write to
-    buffer_to_write_ = nullptr;
+    hand_filled_buffer_ = nullptr;
```

Comment:
```suggestion
    filled_buffer_ = nullptr;
```
Rename hand_filled_buffer. Bad choice